maat-court-data-api/src/main/java/gov/uk/courtdata/link/service/CreateLinkCpJobStatusListener.java

```diff
@@ -23,7 +24,9 @@ public class CreateLinkCpJobStatusListener {
     private final Gson gson;
     private final QueueMessageLogService queueMessageLogService;
 
-    @JmsListener(destination = "${cloud-platform.aws.sqs.queue.createLinkCpStatusJob}")
+//This queue is not in use anymore - May 2023
+@SqsListener(value = "${cloud-platform.aws.sqs.queue.createLinkCpStatusJob}",
```

Comment:
Can we delete this if it is not in use? please verify.

Reply:
We need to double-check this and if we decided to remove then need to update the terraform script on the cloud platform,

 however, I am going to leave this for now.

Reply:
So mark this as resolved.